### DISCUSSION TITLE
Hiding DEBUG level logs?

### DISCUSSION THREAD

**Discussion by PoshoDev** [zauberzeug/nicegui#2275] [Q&A]
### Question

Hello there, 🙂

How could I hide DEBUG level logs produced by `niceGUI`? Ever since I started using the `logging` module on my project, I've started seeing a bunch of debug output logs. I already tried applying a filter but it works on all my other modules except `niceGUI`.

I noticed there's a `uvicorn_logging_level` parameter on `ui.run()`, but that's already set to `"warning"` by default, so I don't know what else to try.

Pardon me if this is a novice question to ask. Hope someone can help me out.

Happy New Year, by the way! ✌

**Comment by stepheweffie** [comment]
Set logging level for niceGUI - 
[https://docs.python.org/3/library/logging.html?highlight=logging#module-logging](url)
`logging.getLogger('niceGUI').setLevel(logging.WARNING)`

**Reply by PoshoDev** [reply]
Hello, 🙂

Thank you very much for your response. Actually, that is exactly what I tried before posting my question (I reckon, I should've shown some code beforehand). Here's a minimum reproducible code for what I'm trying to do.

```python
import logging
from nicegui import ui

logging.getLogger('niceGUI').setLevel(logging.WARNING)

ui.label("Hello World!")

logging.debug("This is a test DEBUG log.")
logging.info("This is a test INFO log.")
logging.warning("This is a test WARNING log.")
logging.error("This is a test ERROR log.")
logging.critical("This is a test CRITICAL log.")

ui.run(port=5500)
```

Using `logging.getLogger('niceGUI').setLevel(logging.WARNING)` blocks any logging level below `WARNING`, including niceGUI's as well as my own project's logs.
```
WARNING:root:This is a test WARNING log.
ERROR:root:This is a test ERROR log.
CRITICAL:root:This is a test CRITICAL log
```

What I was originally trying to ask is, how can I hide/block/ignore niceGUI's `DEBUG` logs without hiding my own? 🤔

**Reply by falkoschindler** [reply]
@PoshoDev Can you show an example for a DEBUG message from NiceGUI? I'm not aware of any such messages.
